CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 3.7.5
+- Move test dependencies back to carthage
+    https://github.com/RevenueCat/purchases-ios/pull/371
+    https://github.com/RevenueCat/purchases-ios/pull/373
+- fixed tests for iOS < 12.2
+    https://github.com/RevenueCat/purchases-ios/pull/372
+- Make cocoapods linking dynamic again
+    https://github.com/RevenueCat/purchases-ios/pull/374
+
+## 3.7.4
+- Fix parsing of dates in receipts with milliseconds
+    https://github.com/RevenueCat/purchases-ios/pull/367
+- Add jitter and extra cache for background processes
+    https://github.com/RevenueCat/purchases-ios/pull/366
+- Skip install to fix archives with direct integration
+    https://github.com/RevenueCat/purchases-ios/pull/364
+
+## 3.7.3
+- Renames files with names that caused issues when building on Windows
+    https://github.com/RevenueCat/purchases-ios/pull/362
+- Fixes crash when parsing receipts with an unexpected number of internal containers in an IAP ASN.1 Container
+    https://github.com/RevenueCat/purchases-ios/pull/360
+- Fixes crash when sending `NSNull` attributes to `addAttributionData:fromNetwork:`
+    https://github.com/RevenueCat/purchases-ios/pull/359
+- Added starter string constants file for logging
+    https://github.com/RevenueCat/purchases-ios/pull/339
+
```

Comment:
added the changes for some of the cherry-picked releases, which weren't directly integrated into develop and so their changelogs didn't make it into develop